app/pathogen/arbovirus/wip/page.tsx

```diff
@@ -0,0 +1,39 @@
+"use client"
+import { useScrollSectionGroup } from "@/components/ui/scroll-section-group/use-scroll-section-group";
+import { notFound, useSearchParams } from "next/navigation";
+import React, { useEffect } from "react";
+import { redesignedArbovirusPageSections, sectionUrlParameterToSectionId } from "./sections";
+
+export default function RedesignedArbovirusPage() {
+  const searchParams = useSearchParams();
+
+  const { renderScrollSectionGroup, moveScrollSectionGroupToSection } = useScrollSectionGroup({
+    scrollSectionGroupProps: {
+      sections: redesignedArbovirusPageSections,
+      className:"col-span-10 row-span-2",
+      scrollThrottleThresholdMilliseconds: 700
```

Comment:
Magic number but basically got this experimentally. If it's too low, you scroll multiple panels at once. If it's too high, you're allowed to scroll freely for a bit after changing panels which I tried to minimize to the best of my ability

400 - way too low
1000 - way too high
700 - works okay

![Peek 2024-01-21 16-23](https://github.com/PathoTracker/Pathotracker/assets/86806388/70bb22cb-cc93-4338-9564-ec152ca99fe9)

Reply:
Actually I turn this to 1000 later because it was driving me nuts when it would scroll multiple panels at once.